scripts/setup/security.ps1

```diff
@@ -153,14 +162,21 @@ function CreateLocalGroup($_name, $desc) {
 
 # Deletes a local group by name.
 # Name: The name of the local group to delete.
-function RemoveLocalGroup($_name) {
+function RemoveLocalGroup($_name, $verifyInstallerFlag) {
 
 	if ([System.String]::IsNullOrEmpty($_name)) {
 		throw "Name cannot be null"
-	}
+    }
 
     $g = GetLocalGroup $_name
 
+    $installerFlag = .\globals.ps1 'INSTALLER_FLAG'
```

Comment:
This logic is polluting the remove localgroup method. The caller should do any verification of the group and then call RemoveLocalGroup.

Reply:
Moving

Reply:
When I put it there I didn't think of it as polluting this method. Any caller of this method can take advantage of the description check logic. However, moving it is fine by me too

Reply:
The idea is that any other invocation can take advantage of the installer flag check by opting in. This can be think of as adding a functionality.

Reply:
RemoveLocalGroup doesn't know anything about installer flags. It has one purpose: remove a local group. It should only have parameters to do that.

Reply:
Moved